src/components/CreateWallet.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <main class="main pb-4">
+    <div class="container-fluid top-bar"></div>
+    <div class="container mt-5">
+      <section class="top-column column">
+        <h1 class="welcome-user">Créer mon portefeuille</h1>
+      </section>
+      <!-- conteneur des colonnes de gauche et droite où se situent les cards -->
+      <!--<div class="column is-flex is-justify-content-space-between pl-0 pr-0">
+        <LeftColAdmin />
+        <RightColAdmin />
+      </div> -->
+    </div>
+  </main>
+</template>
+
+<script lang="ts">
+
+import {defineComponent} from "vue"
```

Comment:
It could be nice to settle on one standard for the syntax, even if this standard is not the same per package, it should at least be consistent in one package itself. I'm not sure this is a real concern about this commit, but if you can have a look for 5mn to use `eslint` (which is available already in the deps) to fix your syntax automatically when you type, that's great, but don't loose more time on it though, there are more important stuff to do right now.

Reply:
Could you please clarify what is wrong in the selected lines? I installed `eslint` on my Visual Studio, and it doesn't detect any issue here. Thks !